rules-tests/Php55/Rector/String_/StringClassNameToClassConstantRector/Fixture/skip_nette_class.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\Php55\Rector\String_\StringClassNameToClassConstantRector\Fixture;
+
+final class SkipNetteClass
+{
+    public function run()
+    {
+        return 'Nette\Utils\FileSystem';
```

Comment:
the configured test is using `Nette\*`, this is new fixture to test it :)

https://github.com/rectorphp/rector-src/blob/9f29bf8434e78a86efe97a08ef3e84c8db2f4a94/rules-tests/Php55/Rector/String_/StringClassNameToClassConstantRector/config/configured_rule.php#L10